docs/compliance/data_privacy_for_mobile.md

```diff
@@ -0,0 +1,151 @@
+- systemVersion: The iOS version
+- systemName: The system name (iOS)
+
+**Android SDK collects:**
+- appIdentifier: Your app's package name
+- appVersion: Your app's version name
+- deviceModel: The device model (e.g., Pixel 7)
+- deviceOS: The operating system (Android)
+- locale: The user's locale
+- language: The user's language
+- sdkType: The type of SDK (android-client)
+- sdkVersion: The version of the Statsig SDK
+- sessionID: A randomly generated UUID for the current session
+- stableID: A persistent device identifier (see below)
+- systemVersion: The Android API level
+- systemName: The system name (Android)
+
+### How can I limit the metadata collected by Statsig?
+
+Both iOS and Android SDKs provide the `optOutNonSdkMetadata` option to limit the collection of device-specific information:
+
+**iOS SDK:**
+```swift
+let user = StatsigUser(userID: "user-123", optOutNonSdkMetadata: true)
+```
+
+**Android SDK:**
+```kotlin
+val options = StatsigOptions(optOutNonSdkMetadata = true)
+```
+
+When `optOutNonSdkMetadata` is enabled, only the following core SDK metadata is included:
+- sdkType: The type of SDK
+- sdkVersion: The version of the Statsig SDK
+- sessionID: A randomly generated UUID for the current session
+- stableID: A persistent device identifier
+
+All device-specific information (appIdentifier, appVersion, deviceModel, deviceOS, locale, language, systemVersion, systemName) is excluded from logs.
+
+### How does StableID work?
+
+The StableID is a persistent identifier that Statsig uses to provide consistent user experiences and analytics:
+
+**iOS Implementation:**
+- The StableID is stored in UserDefaults with the key "com.Statsig.InternalStore.stableIDKey"
+- When first generated, it's created as a random UUID
+- It persists across app launches but is regenerated when the app is reinstalled
+- It can be overridden via the StatsigOptions.overrideStableID parameter
+
+**Android Implementation:**
+- The StableID is stored in SharedPreferences
+- When first generated, it's created as a random UUID
+- It persists across app launches but is regenerated when the app is reinstalled
+- It can be overridden via the StatsigOptions.overrideStableID parameter
+
+The StableID is not shared across different apps or websites and cannot be used to track users across different applications or platforms.
+
+### How can I prevent sending sensitive user data to Statsig?
+
+Use the `privateAttributes` field for sensitive data that should be used for targeting but not logged:
+
+**iOS SDK:**
+```swift
+let user = StatsigUser(
+    userID: "user-123",
+    email: nil, // Not included at top level to keep private
+    privateAttributes: ["email": "user@example.com"] // Used for evaluation but not logged
+)
+```
+
+**Android SDK:**
+```kotlin
+val user = StatsigUser("user-123")
+user.privateAttributes = mapOf("email" to "user@example.com")
+```
+
+These attributes are only used for evaluation and are removed before sending logs to Statsig servers.
```

Comment:
note that they are still sent to statsig servers for the evaluation at initialization time